faqs/zh/stability-fee.md

```diff
@@ -106,21 +106,21 @@ CDP产生的稳定费可以简单按天估算：
 
 > 稳定费 =\(\(借出的 DAI \* \(1 + 年化稳定费率\)\) ^ \(以天计算的债务时长/365\)\) - 借出的 DAI
 
-为期30天的 10,000 DAI 的债务，稳定费率为5.0%时，:
+为期31天的 10,000 DAI 的债务，稳定费率为5.0%时，:
 
 ```text
-(1000 * (1 + 0.05) ^ (30÷365)) - 1000 = 40.18 DAI
+(1000 * (1 + 0.05) ^ (31/365)) - 1000 = 41.524 DAI
 ```
 
 费率为 10.0%:
 
 ```text
-(1000 * (1 + 0.1) ^ (30÷365)) - 1000 = 78.64 DAI
+(1000 * (1 + 0.10) ^ (31/365)) - 1000 = 81.277 DAI
```

Comment:
Have we all agreed this is the new way of displaying this? The numbers we present in this language are going to be different than all the other FAQs...

Reply:
Let's wait for the resolution of [#109](https://github.com/makerdao/community/pull/94).